tasks/check-data-keys/README.md

```diff
@@ -16,7 +16,7 @@ Currently, `releaseNotes`, and `cdn` are the only supported systems.
 | systems  | The systems to check that all data keys are present for | Yes      | []            |
 
 ## Changes in 0.9.1
-* The task would show failure output if a system with just one key was added (as cdn currently is)
+* Improved POSIX compliance and variable handling in the check-data-keys task.
```

Comment:
This seems wrong. Should be a new changelog entry for the new version.

Reply:
Also, why would we want to improve POSIX compliance? We're totally fine using bash and non-posix things.